unit_auto_5_framework/src/main/java/app/DaoType.java

```diff
@@ -0,0 +1,25 @@
+package app;
+
+public enum DaoType {
+
```

Comment:
DAO is a Data  Access object, using for java-representations of DataBase table, incorrect naming

Reply:
renamed to BrowserTypes